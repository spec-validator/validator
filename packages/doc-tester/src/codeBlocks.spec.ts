import fs from 'fs'

import extractCodeBlocks from './codeBlocks'



test('extractCodeBlocks', () => {
  expect(extractCodeBlocks(
    fs.readFileSync(`${__dirname}/fixtures/README.md`).toString(),
    ['present', 'other-present']
  )).toMatchSnapshot()
})
