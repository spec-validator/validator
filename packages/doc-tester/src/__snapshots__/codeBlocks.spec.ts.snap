// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`extractCodeBlocks 1`] = `
"Block 1

Block 2
"
`;
